runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerConfig.java

```diff
@@ -49,6 +49,26 @@ default String getIgnoredItems()
 	)
 	void setIgnoredItems(String key);
 
+	@ConfigItem(
+		keyName = "preferHA",
+		name = "Prefer High Alchemy prices",
+		description = "Show High Alchemy prices for the box total (both prices are still available in the tooltip)"
+	)
+	default boolean preferHA()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "showPriceType",
+		name = "Show Price Type",
+		description = "Whether to show a GE: or HA: next to the total values in the tracker"
+	)
+	default boolean showPriceType()
```

Comment:
I dont really think this is necessary

Reply:
I made this an option so people who didn't want it don't have to use it.

Reply:
Well it is not adding any visual clutter and there is plenty space on loot tracker so its pretty pointless, but sure I guess